mmv1/third_party/terraform/services/storagetransfer/resource_storage_transfer_job.go.erb

```diff
@@ -394,12 +395,70 @@ func transferOptionsSchema() *schema.Schema {
 					ValidateFunc: validation.StringInSlice([]string{"DIFFERENT", "NEVER", "ALWAYS"}, false),
 					Description:  `When to overwrite objects that already exist in the sink. If not set, overwrite behavior is determined by overwriteObjectsAlreadyExistingInSink.`,
 				},
+				"metadata_options": metadataOptionsSchema(),
 			},
 		},
 		Description: `Characteristics of how to treat files from datasource and sink during job. If the option delete_objects_unique_in_sink is true, object conditions based on objects' last_modification_time are ignored and do not exclude objects in a data source or a data sink.`,
 	}
 }
 
+func metadataOptionsSchema() *schema.Schema {
+return &schema.Schema{
```

Comment:
Is this field updatable?

Reply:
Could you please specify what exactly you mean? I believe you are talking about returning it as a pointer vs full struct?

Reply:
Added another comment to be more specific about update tests